docs/gpa.md

```diff
@@ -12,7 +12,7 @@ This document annotates selected parts of the GPA, with details on how to publis
 ```eval_rst
 .. warning::
 
-   References from Articles VII:4, VII:5, IX:8, IX:9, X:7, X:9, X:11, XIII:2, XIV:1, XVI:2 are not yet added.
+   References from Articles XIII:2, XIV:1, XVI:2 are not yet added.
```

Comment:
Did we do IX:8 and IX:9?

Reply:
Ah no, there aren't in the file. I will add them (if relevant).

Reply:
I created an issue. PR is approved!

Reply:
Yes, I'll work on #22. I'll try to suggest guidance.